cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -35,6 +35,19 @@ std::unique_ptr<table> drop_nulls(
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource(),
   cudaStream_t stream                 = 0);
 
+/**
+ * @copydoc cudf::drop_nans(table_view const&, std::vector<size_type> const&,
+ *                                         cudf::size_type, rmm::mr::device_memory_resource*)
```

Comment:
Indentation alignment off?

Reply:
It was same for drop_nans so left as it is. Will try to align and see if clang would be happy.

Reply:
It's inside a comment. Clang-format doesn't do alignment inside comments.